velox/substrait/tests/TypeUtilsTest.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/substrait/TypeUtils.h"
+#include "velox/common/base/tests/GTestUtils.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::substrait;
+
+namespace facebook::velox::substrait::test {
+
+class TypeUtilsTest : public ::testing::Test {
+ protected:
+  static void testToSubstraitType(
+      const TypePtr& type,
+      const std::string& expectedType) {
+    SCOPED_TRACE(type->toString());
+    auto substraitType = substraitSignature(type);
```

Comment:
Do we have tests for toVeloxType function? If not, let's add these. Perhaps, write a round trip test.

Reply:
Already have test for toVeloxType  function [here](https://github.com/facebookincubator/velox/blob/main/velox/substrait/tests/VeloxToSubstraitTypeTest.cpp#L36)

toSubstraitSignature is not an opposite of toVeloxType